src/centralserver/admin-service/src/main/resources/openapi-definition.yaml

```diff
@@ -1,512 +1,4021 @@
+        - description: an optional search string (member name)
+          in: query
+          name: name
+          required: false
+          schema:
+            format: text
+            maxLength: 255
+            minLength: 0
+            type: string
+        - description: an optional search string (instance)
+          in: query
+          name: instance
+          required: false
+          schema:
+            format: text
+            maxLength: 255
+            minLength: 0
+            type: string
+        - description: an optional search string (member_class)
+          in: query
+          name: member_class
+          required: false
+          schema:
+            format: text
+            maxLength: 255
+            minLength: 0
+            type: string
+        - description: an optional search string (member_code)
+          in: query
+          name: member_code
+          required: false
+          schema:
+            format: text
+            maxLength: 255
+            minLength: 0
+            type: string
+        - description: an optional search string (subsystem_code)
+          in: query
+          name: subsystem_code
+          required: false
+          schema:
+            format: text
+            maxLength: 255
+            minLength: 0
+            type: string
+        - description: an optional parameter limiting search to either members or subsystems. If not used, both are searched for
+          in: query
+          name: client_type
+          required: false
+          schema:
+            $ref: '#/components/schemas/ClientType'
+        - description: a security server id – only return clients that belong to this security server
+          in: query
+          name: security_server
+          required: false
+          schema:
+            description: <instance_id>:<member_class>:<member_code>:<security_server_code>
+            example: FI:GOV:123:sserver1
+            format: text
+            type: string
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PagedClients'
+          description: paged clients results
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '403':
+          $ref: '#/components/responses/Forbidden'
+        '500':
+          $ref: '#/components/responses/InternalServerError'
+      summary: find clients
+      tags:
+        - clients
+    post:
+      description:
+        <h3>CS administrator creates a new client (member or subsystem).</h3>
+        <p>The member id will be validated and if the validation fails a validation error is returned.</p>
+        <p>Client's association with security servers is created through management requests.</p>
+      operationId: addClient
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/Client'
```

Comment:
That is probably because `instance_id` is defined as `readOnly: true` (in `MemberId` schema object). That has roots in proxy-ui-api, where `Client.instance_id` is also readOnly. 

Logic for being readOnly is probably that we will only be generating clients for current instance, so it does not make sense to include field for instance_id, which the API user could really only use to set a) the one and only correct value (current instance) b) a wrong value which needs to be validated and results in error.

Reply:
OK